docs/cli/migration/list-nullability.md

```diff
@@ -42,32 +42,32 @@ Developers building an iOS app with Amplify DataStore or Amplify API generates S
 
 ```swift
 public struct ListStringContainer: Model {
-  public var stringList: [String]
-  public var stringNullableList: [String]
+  public var requiredElementRequiredList: [String]
+  public var requiredElementOptionalList: [String]
   public var nullableStringList: [String]?
```

Comment:
```suggestion
  public var optionalElementRequiredList: [String]?
```

Reply:
that's odd, the current file already shows this change